src/style.css

```diff
@@ -0,0 +1,97 @@
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir.ttc");
+}
```

Comment:
add also font-style and font-weight
![Screenshot from 2023-04-01 15-11-12](https://user-images.githubusercontent.com/33029864/229287986-e006bcde-bc0f-45bc-be94-d5688f6f35e2.png)

Reply:
+